tests/network/vmi_multus.go

```diff
@@ -1022,6 +1022,59 @@ var _ = Describe("[Serial]SRIOV", func() {
 				expectInterfaceToExistByMac(virtClient, vmi, mac, hotplugSRIOVDeviceTimeout,
 					"SR-IOV VF is expected to exist in the guest after migration")
 			})
+
+			Context("when aborted", func() {
```

Comment:
You can use the `When` keyword/function instead of `Context`.

Reply:
Done